deepchecks/tabular/checks/distribution/train_test_label_drift.py

```diff
@@ -49,6 +55,8 @@ class TrainTestLabelDrift(TrainTestCheck):
         - 'train_largest': Show the largest train categories.
         - 'test_largest': Show the largest test categories.
         - 'largest_difference': Show the largest difference between categories.
+    categorical_drift_method: str, default: "Cramer"
```

Comment:
Consulted with Bressler, let's change to "cramers_v"?

Reply:
earth_movers has naming with _

Reply:
What?